README.md

```diff
@@ -231,13 +247,28 @@ You can run some unit tests with:
 npm test
 ```
 
-The supervisor runs on Node v10.19.0, so using that specific
+The supervisor runs on Node v12.16.2, so using that specific
 version will ensure tests run in the same environment as
 production.
 
 Alternatively, tests will be run when building the image,
 which ensures that the environment is exactly the same.
 
+## Troubleshooting
+
+Make sure you are running atleast:
+
+```sh
+node -v       # >= 12.16.2
+npm -v        # >= 6.14.0
+docker -v     # >= 19.03.0
+git --version # >= 2.13.0
+```
+
+If you experience errors running any of the steps mentioned here and, don't know how to proceed, try deleting your `node_modules` folder and reinstalling the packages.
```

Comment:
I'd really prefer this was not in the README

Reply:
It was a technique that resolved weird issues for me twice now so I thought it was worth adding but I understand it's a weird tip for troubleshooting.

Reply:
The reason I'm not keen on adding it is it implies that we don't control our module versions, but this isn't true, and in fact I think maybe it's worth rewording it to something like "ensure you use `npm ci`" or something as I expect this is the issue that you were hitting.

Reply:
I've changed the wording for this to remind people to use npm ci instead of deleting modules folder.